Makefile

```diff
@@ -29,5 +27,5 @@ upgrade:
 	helm upgrade connaisseur helm  -n $(NAMESPACE) --wait
 
 annihilate:
-	kubectl delete all,mutatingwebhookconfigurations,clusterroles,clusterrolebindings,configmaps,imagepolicies,secrets,serviceaccounts,crds -lapp.kubernetes.io/instance=connaisseur
+	kubectl delete all,mutatingwebhookconfigurations,clusterroles,clusterrolebindings,configmaps,secrets,serviceaccounts -lapp.kubernetes.io/instance=connaisseur
```

Comment:
I presume it doesn't hurt to keep it in and on the other hand allows to remove the remaining resources of earlier releases. If it does not cause an error, I'd just keep it.